tests/core/service/application/application_service_test.py

```diff
@@ -145,14 +145,16 @@ async def test_update_application_entitlements(app_entitlement_api, application_
     )
 
     assert len(pod_app_entitlements) == 1
-    assert pod_app_entitlements[0].appId == "rsa-app-auth-example"
+    assert pod_app_entitlements[0].app_id == "rsa-app-auth-example"
 
 
+@pytest.mark.skip(
```

Comment:
This test is skipped because the payload used to test it raises an error with the generated model.

UserAppEntitlement -> ProductList -> Product -> `sku` is required 

The payload is the same in the [documentation](https://developers.symphony.com/restapi/reference#user-apps) and the [java BDK](https://github.com/finos/symphony-bdk-java/blob/main/symphony-bdk-core/src/test/resources/application/user_apps.json) and one of the products doesn't have the `sku` attribute.

In python, the [Product](https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/2.0/symphony/bdk/gen/pod_model/product.py) model takes `sku` as a required argument. [Api spec -> product](https://github.com/symphonyoss/symphony-api-spec/blob/master/pod/pod-api-public.yaml#L5353) describes `sku` as required 

Solution: Add the "sku" in the test payload. Does that mean the documentation is wrong? Do we care? How to update it?

Reply:
@symphony-youri Do you have any clue ? IMHO documentation and swagger specs should be consistent with actual behavior

Reply:
I'd say we should update the spec to make it non required, this needs to be done in SBE and in swagger spec repos